federation-js/src/composition/validate/postComposition/__tests__/executableDirectivesIdentical.test.ts

```diff
@@ -81,6 +83,12 @@ describe('executableDirectivesIdentical', () => {
       Array [
         Object {
           "code": "EXECUTABLE_DIRECTIVES_IDENTICAL",
+          "locations": Array [
+            Object {
+              "column": 1,
+              "line": 2,
+            },
+          ],
```

Comment:
Should there be a `locations` entry for each service which defines this directive?